### ISSUE TITLE
StateNavigationButton rendering issue [Blocked on #171]

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#508]
As soon as we open an exploration the StateNavigationButton seems to get rendered before the content and interaction which looks odd.
For checking this issue, open `What is a Fraction?` exploration and you will notice that `Continue` button gets displayed on top-right corner area of screen for a split second, after which everything becomes normal.

**Comment by BenHenning** [comment]
This happens even with #495, and it may be an Espresso rendering issue but we should investigate.

**Comment by rt4914** [comment]
@veena14cs This is not a priority right now.

**Comment by veena14cs** [comment]
Ok unassigning myself.

**Comment by anandwana001** [comment]
PR #171 will solve this issue.

**Comment by anandwana001** [comment]
@rt4914 This issue can be resolve with this pr - https://github.com/oppia/oppia-android/pull/1359

**Comment by rt4914** [comment]
Closing via https://github.com/oppia/oppia-android/pull/1359

**Comment by BenHenning** [comment]
Reopened due to revert: https://github.com/oppia/oppia-android/pull/1627.

**Comment by rt4914** [comment]
Note: This is blocked on #171 which is part of `Beta` and therefore this should be transferred to `Beta` or #171 should be transferred to `Alpha`. @BenHenning Any thoughts?

**Comment by BenHenning** [comment]
Agreed, let's move to beta.

**Comment by EskAlam06** [comment]
This can be closed as now ,  Continue button does not get displayed on top-right corner area of screen for a split second.

Steps-

1. Open What is a Fraction?
2. Observe

Device: Vivo v21 Android 13 version

Timestamp( 1s-12s):-

https://github.com/user-attachments/assets/1fff2b3e-df15-40ac-8d87-e93ccad197fd
